frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices-mode/install-wizard-steps/storage-and-nodes-step.tsx

```diff
@@ -190,14 +190,18 @@ export const StorageAndNodes: React.FC<StorageAndNodesProps> = ({ state, dispatc
           />
         </GridItem>
         <GridItem span={10} className="ocs-install-wizard__select-nodes">
-          <ListPage
-            kind={NodeModel.kind}
-            showTitle={false}
-            ListComponent={AttachedDevicesNodeTable}
-            hideLabelFilter
-            hideNameLabelFilters
-            customData={{ filteredNodes: scNodeNames, setNodes, nodes }}
-          />
+          {scNodeNames.length === 0 ? (
```

Comment:
Don't it needs to be `scNodeNames.length === nodes_selected_on_lvset` ?

Reply:
Maybe I'm missing something, but what I was trying here was - if no nodes are available (that is ZERO nodes) then print the `PVsLoading` message else list the nodes.

Reply:
It may happen that a few PVs wont come up. The nodes found i.e `scNodeNames.length === (state.lvsIsSelectNodes ? state.lvsSelectNodes : state.lvsAllNodes).length`

Reply:
@aruniiird ^^